e2e/testsuite/testconfig.go

```diff
@@ -341,7 +341,44 @@ func fromFile() (TestConfig, bool) {
 		panic(err)
 	}
 
-	return tc, true
+	return populateDefaults(tc), true
+}
+
+// populateDefaults populates default values for the test config if
+// certain required fields are not specified.
+func populateDefaults(tc TestConfig) TestConfig {
+	for i := range tc.ChainConfigs {
+		if tc.ChainConfigs[i].ChainID == "" {
+			tc.ChainConfigs[i].ChainID = fmt.Sprintf("chain-%d", i+1)
+		}
+		if tc.ChainConfigs[i].Binary == "" {
+			tc.ChainConfigs[i].Binary = defaultBinary
+		}
+		if tc.ChainConfigs[i].Image == "" {
+			tc.ChainConfigs[i].Image = getChainImage(tc.ChainConfigs[i].Binary)
+		}
+		if tc.ChainConfigs[i].NumValidators == 0 {
+			tc.ChainConfigs[i].NumValidators = 1
+		}
+	}
+
+	if tc.ActiveRelayer == "" {
+		tc.ActiveRelayer = relayer.Hermes
+	}
+
+	if tc.RelayerConfigs == nil {
+		tc.RelayerConfigs = []relayer.Config{
+			getDefaultRlyRelayerConfig(),
+			getDefaultHermesRelayerConfig(),
+			getDefaultHyperspaceRelayerConfig(),
+		}
+	}
+
+	if tc.CometBFTConfig.LogLevel == "" {
+		tc.CometBFTConfig.LogLevel = "info"
+	}
+
+	return tc
```

Comment:
enables the minimal config, Happy to move this to a separate PR if people would prefer.